src/cpp/session/modules/rmarkdown/NotebookAlternateEngines.R

```diff
@@ -54,8 +54,47 @@
       stop(sprintf(fmt, options))
    }
    
-   # invoke engine
+   # prepare the R environment for reticulate Python engine
+   if ("reticulate" %in% loadedNamespaces())
```

Comment:
How does the `reticulate` namespace get loaded? (I'm thinking of the case where you restart R, and as your first gesture run a Reticulate chunk with a plot.)

Reply:
Nice catch; indeed, we need to make sure `reticulate` is loaded here if possible. I'll make that change.